storm-core/test/clj/backtype/storm/nimbus_test.clj

```diff
@@ -846,16 +846,6 @@
                   NIMBUS-SLOTS-PER-TOPOLOGY 8}]
     (letlocals
       (bind topology (thrift/mk-topology
-                        {"1" (thrift/mk-spout-spec (TestPlannerSpout. true) :parallelism-hint 0 :conf {TOPOLOGY-TASKS 1})}
```

Comment:
Because thrift/mk-topology actually throws IllegalArgumentException, not submit-local-topology.
We should wrap bind statement to check IllegalArgumentException.